db/models.js

```diff
@@ -87,4 +87,32 @@ exports.fetchCommentsById = (id) => {
           }
         return comments.rows;
     });
-}
\ No newline at end of file
+}
+
+exports.insertComment = (id, body, author) => {
+  if(isNaN(id)) {
+    return Promise.reject({
+      status: 400,
+      msg: `Incorrect data type`,
+    });
+  } else if(!body || !author) {
+    return Promise.reject({
+      status: 400,
+      msg: `Post body incorrect`,
+    });
+  } 
+return db.query(`SELECT articles.article_id FROM articles`).then((article_id)=>{
+  if(id > article_id.rows.length){
```

Comment:
Using the length is a bad way to check if the id exists. I suggest making a separate query with the id that's being passed in.